src/main/java/use_case/notification/NotificationCreationBoundary.java

```diff
@@ -0,0 +1,35 @@
+package main.java.use_case.notification;
+
+import main.java.constants.NotificationType;
+
+import java.time.Duration;
+
+public class NotificationCreationBoundary {
```

Comment:
What is the purpose of this class? This class is not used by the controllers

Reply:
It's being used as a DTO in the NotificationController, lines 52 and 63.
The name doesn't line up with the purpose, though. At least not by the convention previously set. See On CreationBoundaries section of my comment above

Reply:
I changed the code to be consistent with the DTOs are you have mentioned